style.css

```diff
@@ -146,426 +370,507 @@ button[type="submit"] {
+  background-repeat: no-repeat;
+  background-size: cover;
+  width: 100%;
 }
 
-.menu-2 {
-  color: #7a869a;
+.contact-section form h3 {
+  font-size: 40px;
+  text-align: center;
+  color: #fff;
+  font-weight: 700;
 }
 
-.menu ul li {
-  display: inline-block;
-  padding: 0 5px;
+.contact-section form p :hover {
+  transform: translate(0, 10px);
+  font-weight: 700;
 }
 
-p {
-  color: #344563;
-  font-family: "Poppins", sans-serif;
+.contact-form {
+  display: flex;
+  flex-direction: column;
+  margin: 67px 0;
+  padding: 0;
 }
 
-.knowledges {
+/* popup */
+.card-popup-window {
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
   width: 100%;
-  display: flex;
+  background-color: rgba(169, 173, 175, 0.9);
+  z-index: 999;
+  display: none;
+  overflow-y: auto;
+}
+
+.popup-main-container {
+  width: 100%;
+  height: auto;
   justify-content: center;
-  align-items: center;
-  flex-direction: column;
+  display: flex;
 }
 
-.knowledges .knw {
-  width: 90%;
-  margin: 0 auto;
+.card-popup-window.open-pop-up {
+  display: block;
+}
+
+.popup-main-container .main-container {
+  width: 80%;
+  height: auto;
   display: flex;
   justify-content: center;
-  align-items: center;
   flex-direction: column;
+  background-color: #fff;
+  border-radius: 16px;
+  border: 1px solid #dfe1e6;
+  padding: 2rem;
+  position: relative;
+  z-index: 9999;
+  margin: 3rem 0;
 }
 
-.btn-T button {
-  border: 1px solid #dfe1e6;
-  padding: 10px 20px;
-  outline: none;
-  transition: padding 3s;
-  background-color: transparent;
-  color: #396df5;
-  border-radius: 12px;
-  box-sizing: border-box;
-  margin: 22px 0;
-  font-family: "Poppins", sans-serif;
-  font-weight: 500;
-  font-size: 18px;
-  border-color: #7f8cff;
+.card-popup-window .main-container .title {
+  margin-bottom: 0.5rem;
 }
 
-.btn-T button:focus {
-  background-color: #2230d2;
-  color: #fff;
+.card-popup-window .main-container .title h2 {
+  font-size: 28px;
+  font-weight: 700;
+  color: #172b4d;
 }
 
-.btn-T button:hover {
-  background-color: #4053fc;
-  padding-left: 40px;
-  color: #fff;
+.card-popup-window .main-container .info {
+  margin-bottom: 2rem;
+  height: auto;
+  color: #7a869a;
 }
 
-.aboutMe {
+.card-popup-window .main-container .info span:first-child {
+  color: #000;
+  font-weight: 600;
+}
+
+.card-popup-window .main-container .portfolio-img {
   width: 100%;
-  padding-top: 5rem;
-  background-color: #fff;
-  padding-left: 12px;
+  text-align: center;
 }
 
-.knowledges .knw .lang {
+.card-popup-window .main-container .portfolio-img img {
+  max-width: 100%;
+  width: 100%;
+}
+
+.left-container {
   width: 100%;
   display: flex;
-  align-items: center;
-  justify-content: space-between;
-  padding-bottom: 1.5rem;
-  position: relative;
+  flex-direction: column;
+}
+
+.card-popup-window .left-container .detail {
+  margin-top: 5px;
+  text-align: start;
+}
+
+.card-popup-window .main-container .languages {
+  display: flex;
+  margin-top: 2rem;
 }
```

Comment:
- [ ] <b>REQUIRED:</b> As the previous reviewer suggested, this content needs a little fix to look exactly like the provided design.
   
| How it is currently is      | How it is supposed to be |
  | ----------- | ----------- |
| ![scrnli_9_4_2021_8-09-46 PM](https://user-images.githubusercontent.com/45217364/132104367-8cd7f833-a697-4911-9e2f-44db6d28952d.png) | ![scrnli_9_4_2021_8-10-33 PM](https://user-images.githubusercontent.com/45217364/132104371-355f7237-7c0a-4430-91b2-7055565f79d8.png) |

To fix this, just add this rule `flex-wrap: wrap` in the above-highlighted code.